src/component/selection/getSelectionOffsetKeyForNode.js

```diff
@@ -17,7 +17,15 @@
  * Get offset key from a node.
  */
 function getSelectionOffsetKeyForNode(node: Node): ?string {
-  return node instanceof Element ? node.getAttribute('data-offset-key') : null;
+  if (node instanceof Element) {
+    var offsetKey = node.getAttribute('data-offset-key');
+    if (offsetKey) return offsetKey;
```

Comment:
Styling nit. We should have eslint catch this.

```javascript
if (offsetKey) {
  return offsetKey;
}
```

Same for the `childOffsetKey` case below.